addons/website/static/src/js/editor/snippets.options.js

```diff
@@ -1592,65 +1592,25 @@ options.registry.anchorName = options.Class.extend({
     /**
      * @override
      */
-    onClone: function () {
-        this.$target.removeAttr('id data-anchor');
-    },
-
-    //--------------------------------------------------------------------------
-    // Options
-    //--------------------------------------------------------------------------
+    start: function () {
+        // Initialize tooltip, use a DP to hide it 1500ms after the last click
+        const $button = this.$el.find('we-button');
+        $button.tooltip({title: _t("Copied !"), trigger: "manual", placement: "top"});
+
+        // Generate anchor and copy it to clipboard on click, show the tooltip on success
+        const clipboard = new ClipboardJS($button[0], {text: this._getAnchor.bind(this)});
+        clipboard.on('success', () => {
+            $button.tooltip('show');
+            concurrency.delay(800).then(() => $button.tooltip('hide'));
```

Comment:
Use `concurrency.delay` only when you actually need a `Promise`, otherwise a `setTimeout(() => ..., 800)` is more standard.